airflow_dbt_python/hooks/localfs.py

```diff
@@ -127,33 +126,6 @@ def copy(
 
         copy_function = partial(self.copy_one, replace=replace)
 
-        if sys.version_info.major == 3 and sys.version_info.minor < 8:
-            py37_copytree(source, destination, replace)
-        else:
-            shutil.copytree(  # type: ignore
-                source, destination, copy_function=copy_function, dirs_exist_ok=True
-            )
-
-
-def py37_copytree(source: URL, destination: URL, replace: bool = True):
```

Comment:
Happy to see this workaround removed. Fully behind dropping support for Python 3.7.